scripts/buildCSS.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const CleanCSS = require('clean-css');
+
+const compileCSS = require('react-with-styles-interface-css-compiler');
+
+const registerMaxSpecificity = require('react-with-styles-interface-css/dist/utils/registerMaxSpecificity').default;
+const registerCSSInterfaceWithDefaultTheme = require('../src/utils/registerCSSInterfaceWithDefaultTheme').default;
```

Comment:
hmm - the `.default` is unfortunate. can this pull from `lib` instead of `src` - so it must run after the build process - and then it can require from the babel output, which would have `add-module-exports` applied?

Reply:
Is `add-module-exports` part of the airbnb babel config? We tried to pull from lib and it still needs the `.default` (also notably, `registerMaxSpecificity` needs the `.default` even though it is a built file.